docs/pages/reference/backends.mdx

```diff
@@ -391,6 +391,62 @@ credentials, or [managed
 identity](https://learn.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview)
 credentials.
 
+### Google Cloud IAM authentication
+
+If you are running Teleport on Google Cloud, Teleport can make use of [IAM
+Authentication](https://cloud.google.com/sql/docs/postgres/iam-authentication)
+to connect to an GCP Cloud SQL for PostgreSQL without having to manage any
+secrets:
+```yaml
+teleport:
+  storage:
+    type: postgresql
+    auth_mode: gcp-cloudsql
+
+    # GCP connection name is in format of <project>:<location>:<instance>.
+    gcp_connection_name: <Var name="project:location:instance"/>
+
+    # Type of IP for GCP connections. Valid options are:
```

Comment:
I would make it more explicit that this is the IP type for the user's Cloud SQL instance